crates/bevy_internal/Cargo.toml

```diff
@@ -152,6 +152,9 @@ file_watcher = ["bevy_asset?/file_watcher"]
 # Enables watching embedded files for Bevy Asset hot-reloading
 embedded_watcher = ["bevy_asset?/embedded_watcher"]
 
+# Enable the bevy_ui debug overlay
+bevy_ui_debug = ["bevy_dev_tools?/bevy_ui_debug", "bevy_dev_tools"]
```

Comment:
I did so when someone add this feature to the TOML, it will auto add bevy_dev_tools feature too. I'm not sure if that's a good idea or not.

Reply:
See comment above: we should just cut the top-level feature IMO.